conjure/conjure.go

```diff
@@ -124,6 +125,21 @@ func GenerateOutputFiles(conjureDefinition spec.ConjureDefinition, cfg OutputCon
 			}
 			files = append(files, newGoFile(filepath.Join(pkg.OutputDir, "servers.conjure.go"), serverFile))
 		}
+		if len(def.Extensions) > 0 {
+			extensions := "extensions.conjure.json"
```

Comment:
```suggestion
			const extensionsFileName = "extensions.conjure.json"
```

nit, but prefer the above (either inline as it currently is, or as a package-level variable) since it makes it more semantically clear what the value is and to make it clear that this is a constant rather than a mutable variable

Reply:
1480daf1